src/main/java/it/pagopa/interop/probing/eservice/operations/service/impl/EserviceServiceImpl.java

```diff
@@ -122,17 +123,16 @@ public void updateEserviceFrequency(UpdateEserviceFrequencyDto inputData)
         .pollingStartTime(inputData.getNewPollingStartTime())
         .pollingEndTime(inputData.getNewPollingEndTime());
     eserviceRepository.save(eServiceToUpdate);
-
-    log.info("Eservice " + eServiceToUpdate.eserviceId() + " with version "
-        + eServiceToUpdate.versionId() + " has been updated with startTime: "
-        + eServiceToUpdate.pollingStartTime() + " and endTime: " + eServiceToUpdate.pollingEndTime()
-        + " and frequency: " + eServiceToUpdate.pollingFrequency());
+    logger.logMessageEservicePollingConfigUpdated(eServiceToUpdate.eserviceId(), eServiceToUpdate.versionId(),
+        eServiceToUpdate.pollingStartTime(), eServiceToUpdate.pollingEndTime(),
+        eServiceToUpdate.pollingFrequency());
   }
 
   @Override
   public SearchEserviceResponse searchEservices(Integer limit, Integer offset, String eserviceName,
       String producerName, Integer versionNumber, List<EserviceMonitorState> state) {
 
+    logger.logMessageSearchEservice(limit, offset, eserviceName, producerName, versionNumber, state);
     Page<EserviceView> eserviceList = null;
```

Comment:
Use empty() to initialize this variable.

Reply:
Directly removed initialization since the Page object is anyway initialized by the following methods invocations hence its initialization results as redundant. Push done.